src/Microsoft.AspNetCore.Mvc.Core/Routing/UrlHelper.cs

```diff
@@ -116,11 +125,12 @@ public virtual string RouteUrl(UrlRouteContext routeContext)
         }
 
         /// <summary>
-        /// Gets the <see cref="VirtualPathData"/> for the specified route values by using the specified route name.
+        /// Gets the <see cref="VirtualPathData"/> for the specified <paramref name="routeName"/> and route
+        /// <paramref name="values"/>.
         /// </summary>
         /// <param name="routeName">The name of the route that is used to generate the <see cref="VirtualPathData"/>.
         /// </param>
-        /// <param name="values">A dictionary that contains the parameters for a route.</param>
+        /// <param name="values">The <see cref="RouteValueDictionary"/>.</param>
```

Comment:
We should say more about this, these values are combined with the ambient values and used by the route to generate the URL.